.github/workflows/connector-performance-command.yml

```diff
@@ -133,15 +142,18 @@ jobs:
           DS: ${{ github.event.inputs.dataset }}
           prefix: '{"type":"LOG","log":{"level":"INFO","message":"INFO i.a.i.p.PerformanceTest(runTest):165'
           suffix: '"}}'
+          HARNESS_TYPE: ${{ steps.which-harness.outputs.harness_type }}
         run: |
           kubectl apply -f ./tools/bin/admin-service-account.yaml
           connector_name=$(echo $CONN | cut -d / -f 2)
           kind load docker-image airbyte/$connector_name:dev --name chart-testing
-          kind load docker-image airbyte/source-harness:dev --name chart-testing
+          kind load docker-image airbyte/$HARNESS_TYPE:dev --name chart-testing
```

Comment:
nit: we are using both upper and lower case here. standardize?

Reply:
Fixed. thanks